dotCMS/src/main/java/com/dotcms/analytics/AnalyticsAPI.java

```diff
@@ -0,0 +1,29 @@
+package com.dotcms.analytics;
+
+
+import com.dotcms.analytics.app.AnalyticsApp;
+import com.dotcms.analytics.model.AccessToken;
+import com.dotcms.analytics.model.AnalyticsKey;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.util.Config;
+
+
+/**
+ * Analytics functionality interface.
+ *
+ * @author vico
+ */
+public interface AnalyticsAPI {
```

Comment:
Integration tests for this API?

Reply:
On its wat.

Reply:
@dsilvam integration tests for this class will depend on having stable docker containers for KeyCloak and Jitsu Configurator.